### PR TITLE
install.md: quick pass on content and structure

### PR DESCRIPTION
**Pull request by matildepark**
I cleaned up the install instructions according to the original intention of being geared "per ID type".

- Removed instructions on "joining a chat" since it no longer applies in new groups world
- Sentence case headers
- Tried to be smarter with links and explanations; often merging sections that were about the same thing
- Streamlined the general flow

| Before                                                                                                                                                                 | After                                                                                                                                                                  |
|------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Screenshot 2020-09-07 at 8 23 22 PM | Screenshot 2020-09-07 at 8 22 42 PM |

### CODE REVIEW

File: content/using/install.md [hunk 1]
```diff
@@ -6,41 +6,25 @@ template = "page_indiced.html"
...
+As we'll explain, there are a few types of Urbit ID, but everyday users tend to use **planets**, as full citizens of the network.
 
-On the network, Urbit IDs function as network addresses, represented by pronounceable names like `~padmyn-pasnux`. There are a few kinds of Urbit ID, but ones meant for typical users are called **planets**. Anyone who has a planet can be thought of as a full "citizen" of the Urbit network. But to run your ship on the network, you need to start it with a piece of cryptographic information from its Urbit ID. This guide walk you through getting onto Urbit with an ID.
+Two other types of ships are **stars** and **galaxies**. Together these perform essential infrastructural roles for the Urbit network. Each galaxy can distribute 256 stars, and each star can distribute 65,536 planets. So any given planet ultimately came from a specific star, which itself came from a specific galaxy. On the Urbit OS side, stars help route packets; galaxies are more like DNS root servers or ICANN members. The difference with Urbit in the analogy is that Urbit IDs are owned cryptographically by many different people and accrue reputation independently.
```
Comment (drbeefsupreme) [inline content/using/install.md]: > is that Urbit IDs are owned cryptographically by many different people 

This sort of sounds like multiple people can own a single ID.

File: content/using/install.md [hunk 2]
```diff
@@ -69,29 +59,15 @@ sudo setcap 'cap_net_bind_service=+ep' /path/to/urbit
...
+> Please note that this method of installing Urbit is experimental, and we may not be able to assist you if you encounter issues related to WSL 2.
 
 These instructions have been tested and verified for WSL 2 + Ubuntu 18.04 LTS.
 
-Urbit cannot run on Windows itself, but there is a convenient way to run a Linux distro using the [Windows Subsystem for Linux 2](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux) on Windows 10. 
-
-For Urbit to work, it is necessary to [install WSL 2](https://docs.microsoft.com/en-us/windows/wsl/wsl2-install) and not just WSL.
-
-Once WSL 2 is installed, open a Linux terminal in Windows and then follow the Linux installation instructions above. 
+Urbit cannot run on Windows itself, but there is a convenient way to run a Linux distro using the [Windows Subsystem for Linux 2](https://docs.microsoft.com/en-us/windows/wsl/wsl2-install) on Windows 10. Install the Windows Subsystem for Linux 2 and open a Linux terminal in Windows, following the Linux installation instructions above.
```
Comment (drbeefsupreme) [inline content/using/install.md]:
```suggestion
Urbit cannot run on Windows itself, but there is a convenient way to run a Linux distro using the [Windows Subsystem for Linux 2](https://docs.microsoft.com/en-us/windows/wsl/wsl2-install) on Windows 10. Install the Windows Subsystem for Linux 2 and open a Linux terminal in Windows, then follow the Linux installation instructions above.
```

File: content/using/install.md [hunk 3]
```diff
@@ -69,29 +59,15 @@ sudo setcap 'cap_net_bind_service=+ep' /path/to/urbit
...
 
 For performance reasons, do not install Urbit in the mounted Windows volume, but install it in the Linux file system. For example, in your home directory, by typing `cd ~`.
```
Comment (drbeefsupreme) [inline content/using/install.md]:
```suggestion
For performance reasons, do not install Urbit in the mounted Windows volume, but install it in the Linux file system. For example, in your home directory, which can be navigated to by entering `cd ~`.
```

File: content/using/install.md [hunk 4]
```diff
@@ -164,39 +139,37 @@ The `>=` output means that a command was successful. Now you can see your ship's
...
-Click that `Chat` type on your Landscape home page, and then click the `Join Chat` column. Then, enter `~bitbet-bolbel/urbit-community` into the field and press the `Join Chat` button.
+Click the 'Groups' button on your Landscape home page, and then click the `Join Group` column. Then, enter `~bitbet-bolbel/urbit-community` into the field and press the `Join Group` button.
 
-You can send and receive messages here, just like any other chatroom.
+From the Urbit Community group you can join a variety of chatrooms, notebooks, and collections.
 
-## Getting an Urbit ID (Planet or Star)
+## Setting up a planet {#id}
+
+If you have an Urbit ID, you'll use the [Bridge](https://bridge.urbit.org) client to get your ship's *keyfile* before you can [boot your ship](#booting-your-ship).  Your keyfile is a cryptographic secret that allows your Urbit instance to authenticate itself on the network, so you can't boot your planet without it. Your keyfile is deterministically derived from your other cryptographic secrets.
```
Comment (drbeefsupreme) [inline content/using/install.md]: I think the way this is written might give some people the idea that you need to use your keyfile for subsequent boots (I know we clarify this later but I generally don't want to assume people read the whole thing)

```suggestion
If you have an Urbit ID, you'll use the [Bridge](https://bridge.urbit.org) client to get your ship's *keyfile* before you can [boot your ship](#booting-your-ship).  Your keyfile is a cryptographic secret that allows your Urbit instance to authenticate itself on the network, so it is required the first time you boot your ship. Your keyfile is deterministically derived from your other cryptographic secrets.
```

**Review by drbeefsupreme — CHANGES_REQUESTED** [review]
Looks good to me besides the few nitpicks, and I strongly appreciate all improvements to the install page!

File: content/using/install.md (hunk 1 above)
Comment (matildepark) [reply content/using/install.md]: Well, they can, but I get the intention, will edit

**Comment by matildepark** [comment]
Ready for another review.

**Review by drbeefsupreme — APPROVED** [review]

**Pull request merged**